go/vt/vtadmin/api.go

```diff
@@ -956,6 +958,45 @@ func (api *API) GetTablet(ctx context.Context, req *vtadminpb.GetTabletRequest)
 	return nil, vterrors.Errorf(vtrpcpb.Code_NOT_FOUND, "%s: %s, searched clusters = %v", errors.ErrAmbiguousTablet, req.Alias, ids)
 }
 
+// PingTablet is part of the vtadminpb.VTAdminServer interface.
+func (api *API) PingTablet(ctx context.Context, req *vtadminpb.PingTabletRequest) (*vtadmin.PingTabletResponse, error) {
+	span, ctx := trace.NewSpan(ctx, "API.PingTablet")
+	defer span.Finish()
+
+	tablet, err := api.GetTablet(ctx, &vtadminpb.GetTabletRequest{
+		Alias:      req.Alias,
+		ClusterIds: req.ClusterIds,
+	})
+	if err != nil {
+		return nil, err
+	}
+
+	c, ok := api.clusterMap[tablet.Cluster.Id]
+	if !ok {
+		return nil, fmt.Errorf("%w: no such cluster %s", errors.ErrUnsupportedCluster, tablet.Cluster.Id)
+	}
+
+	cluster.AnnotateSpan(c, span)
+
+	if !api.authz.IsAuthorized(ctx, c.ID, rbac.VSchemaResource, rbac.GetAction) {
```

Comment:
i think we need a new resource type, but i don't have a good naming suggestion yet 🤔 something that conveys "completely harmless action that isn't really Read-Only or Read-Write"

Reply:
I missed this copy pasta! How bout "PingAction" for now? It's not a traditional http action but follows the naming of PingTablet

Reply:
Otherwise "Get" seems like the http action that best matches 🤔

Reply:
yeah Ping is good here!